core/scripts/pptr-run-devtools.js

```diff
@@ -328,6 +341,16 @@ async function testUrlFromDevtools(url, options = {}) {
     const result = await evaluateInSession(inspectorSession, runLighthouse, [addSniffer]);
 
     return {...result, logs};
+  } catch (err) {
+    if (options.detailedError && inspectorSession) {
```

Comment:
why not always do this?

Reply:
The screenshot URL is very long compared to other output. This isn't really "noise" for the reason you mentioned but it is extra debug information. Since this is for errors and not expectation differences it makes more sense to always show the screenshot.

Reply:
Could it save to disk in `.tmp` and just print the path?

Reply:
Ehh I like having the screenshot easily accessible in CI logs. I don't want to have to download CI failure artifacts if I don't have to.

Plus, this file is organized to be called by a smoke runner but isn't aware of any smoke artifact folders itself.